lib/codecs/src/decoding/format/mod.rs

```diff
@@ -44,3 +46,8 @@ dyn_clone::clone_trait_object!(Deserializer);
 
 /// A `Box` containing a `Deserializer`.
 pub type BoxedDeserializer = Box<dyn Deserializer>;
+
+/// Default value for the UTF-8 lossy option.
+const fn default_lossy() -> bool {
+    true
+}
```

Comment:
I know we have a shared function for [`default_true`](https://github.com/vectordotdev/vector/blob/c97d619d47b1171d592dcf55692b5caa01e97992/src/serde.rs#L11) - it's specifically for `serde`, but maybe could be reused?

Reply:
Yeah, I considered that. The issue is that `default_true()` is defined in `vector`, meaning that we can't import it in `codecs` without a circular dependency. We could move it to `vector_corde::serde` but it didn't seem worth the effort at the time. What do you think?

Reply:
Eh, probably not worth it - I remember wishing before you could more easily `default = bool::True` without so much hassle.